examples/dgmulti_2d/elixir_navier_stokes_convergence.jl

```diff
@@ -0,0 +1,206 @@
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# semidiscretization of the ideal compressible Navier-Stokes equations
+
+get_Re() = 100
+get_Pr() = 0.72
+
+equations = CompressibleEulerEquations2D(1.4)
+# Note: If you change the Navier-Stokes parameters here, also change them in the initial condition
+# I really do not like this structure but it should work for now
```

Comment:
Should we remove this line before merging?

Reply:
Yes we can remove this. I believe that this issue have been addressed.